.github/workflows/openapi-generate.yml

```diff
@@ -0,0 +1,33 @@
+name: Update gem from OpenAPI Spec
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+  - cron: '0 0 * * 0'
+
+jobs:
+  openapi-generate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up openapi-generator-cli
+      run: wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/7.9.0/openapi-generator-cli-7.9.0.jar -O openapi-generator-cli.jar
```

Comment:
curious if there's a way to avoid hardcoding the version number.

Reply:
Yeah agreed, there isn't a latest symlink but there is a metadata.xml with the latest value there so we can use that.

Reply:
quick google shows an API at https://central.sonatype.com, but I think it requires auth, so it might be simpler to just scrape https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/

Reply:
```
xq -x '//metadata/versioning/latest' maven-metadata.xml 
7.12.0
```

Reply:
Done